_includes/filter-tabs.md

```diff
@@ -0,0 +1,15 @@
+{% comment %}
+In order to properly utilize this file, the following three items must be added to the YAML front matter of each page:
```

Comment:
I'll definitely add the note about this being for filters between different URLs rather than showing/hiding DIVs on the same page.

The location of the include was what we discussed on Tuesday, and I think we said we wanted to give users flexibility to place it where we want.